Tests/PLCrashAsyncMachOImageTests.m

```diff
@@ -48,8 +48,9 @@ @implementation PLCrashAsyncMachOImageTests
 - (void) setUp {
     /* Fetch our containing image's dyld info */
     Dl_info info;
-    STAssertTrue(dladdr([self class], &info) > 0, @"Could not fetch dyld info for %p", [self class]);
-
+    void *classRef = (__bridge_retained void *)([self class]);
```

Comment:
No need to transfer ownership here. Why not just `__bridge`? See [an article about it](https://medium.com/@notestomyself/bridge-vs-bridge-transfer-vs-bridge-retained-f84e6b6406d1)